float32 steering_angle
float32 speed
float32 sign_detection